pyaerocom/io/readungridded.py

```diff
@@ -88,32 +88,6 @@ def __init__(self, data_ids=None, ignore_cache=False, data_dirs=None):
             logger.info("Deactivating caching")
             const.CACHING = False
 
-    @property
```

Comment:
Is this really not needed anywhere else in the code?

Reply:
The same property was defined twice, so I removed the 1st definition...